sortinghat/core/recommendations/gender.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014-2021 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     Santiago Dueñas <sduenas@bitergia.com>
+#     Miguel Ángel Fernández <mafesan@bitergia.com>
+#     Eva Millán <evamillan@bitergia.com>
+#
+
+
+import logging
+import re
+
+import requests
+import urllib3.util
+
+from ..db import find_individual_by_uuid
+from ..errors import NotFoundError, InvalidValueError
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+name_pattern = re.compile(r"(^\w+)\s\w+")
+
+api_key = settings.GENDERIZE_API_KEY
```

Comment:
Can we change the name of the setting to `SORTINGHAT_GENDERIZE_API_KEY`? I start thinking we should use `SORTINGHAT_` prefix with all the SortingHat settings to differentiate them from the rest of the settings.